### PR TITLE
Sahil/landing page

### PR DESCRIPTION
**Pull request by sdani1923**
PR for landing page:
- thanks for helping me out with the formatting part of the faq section and the main logo.
- changed the text in the about section to match what it currently is in the Figma. I remember you mentioning on Slack that you wanted to give me new updated info for this section. Let me know what you need there and I can update it any time in a couple of minutes.
- schedule section has been formatted as per the Figma. Also updated a few things in the @media max-width 440px and 900px section to get the formatting proper on smaller screen sizes.
- faq section contains the list of FAQs you sent me over Slack. I was a little confused by what you meant in the "What if I don't know anything about hardware?" question. So, I did it as I understood. Please correct me if I have misinterpreted what you wanted there for that one, and I'll fix that immediately.
- have added the sponsor logos for TI and Qualcomm as per our conversation.

### CODE REVIEW

File: public/css/index.css [hunk 1]
```diff
@@ -140,21 +277,58 @@
...
+
+.question {
+	margin: 12px 0 0;
+	font-size: 22px;
+	font-weight: 700;
+}
+
+.answer {
+	margin: 17px 0 0;
+	font-size: 16px;
```
Comment (robertnperalta) [inline public/css/index.css]: This one is on me, but when a question and/or answer wrap, they end up a little too close to the bottom of the box they're in. I think you should be able to fix this by changing the bottom margin for both to be 10px

File: views/index.hbs [hunk 2]
```diff
@@ -411,32 +466,96 @@
...
+            </div>
+
+            <div class="q-a">
+                <div class="question-section clearfix">
+                    <p class="q">Q.</p>
+                    <p class="question">What if I don't know anything about hardware?</p>
+                </div>
+                <div class="answer-section clearfix">
+                    <p class="a">A.</p>
+                    <p class="answer">You are definitely not alone! During the hackathon, there will be workshops on hardware and hacking that will be accessible for everyone. In addition, our resources page provides quickstart info and basic ideas for specifc parts featured in our <a class='faq-link' href='/boxes'>boxes</a>. Create an account to check out what's in these boxes!</p>
```
Comment (robertnperalta) [inline views/index.hbs]: This one is on me too; I wasn't clear when I sent you the list of questions. They don't need to create an account to view the boxes, it's so they can view the resources page. Also, you should probably put a link to `/dashboard/resources` where you say "resources page". They'll get intercepted by the login page, but it makes it a little clearer we're talking about a page on the website

File: public/css/index.css [hunk 3]
```diff
@@ -140,21 +277,58 @@
...
+.question-section {
+	background-color: rgba(255, 255, 255, 0.5);
+	border-left: 10px solid #ec3fa6;
+	padding-left: 15px;
+}
+
+.answer-section {
+	padding-top: 10px;
+	padding-left: 25px;
+}
```
Comment (robertnperalta) [inline public/css/index.css]: Another detail I missed: add 15px right padding to both of these sections to make the text wrap at a more comfortable distance from the edge of their boxes

**Review by robertnperalta — CHANGES_REQUESTED** [review]
Looks good! Thanks for your work on this. Only a couple things to change this time. One thing I couldn't comment in the code: change all the links on the landing page to be magenta by default, just so they stand out a bit more. The one exception to this should be the sponsorship brochure link, which already has a class overriding the default.

Something else we might consider at a later date is making the sponsor logos larger on big screens so it doesn't look as empty if we can't get other sponsorships. But this isn't something to worry about rn

File: public/css/index.css (hunk 1 above)
Comment (sdani1923) [reply public/css/index.css]: Fixed!

File: public/css/index.css (hunk 3 above)
Comment (sdani1923) [reply public/css/index.css]: Fixed!

File: views/index.hbs (hunk 2 above)
Comment (sdani1923) [reply views/index.hbs]: Fixed!

**Comment by sdani1923** [comment]
Hey, I fixed everything as per your PR comments! Looks all done and good to me now! Please let me know if any further changes/modifications are needed.

**Review by robertnperalta — APPROVED** [review]
Lgtm!

**Pull request merged**